src/markdown-service.ts

```diff
@@ -0,0 +1,91 @@
+import { getComparisonLinksObject } from './compare-service'
+import { ComparisonResultsByURLInterface } from './types.d'
+
+export interface MarkdownTableCellInterface {
+  currentValue: number
+  isRegression: boolean
+  diffValue: number
+  metricUnit: string
+  metricType: string
+}
+
+export const getMarkdownTableCell = ({
+  currentValue,
+  isRegression,
+  diffValue,
+  metricUnit,
+  metricType
+}: MarkdownTableCellInterface): string => {
+  if (metricType === 'performance') {
+    return `[${currentValue}${metricUnit} ${isRegression ? '🔴' : '🟢'}](## "Performance has  ${
+      isRegression ? 'decreased in ' : 'improved in +'
+    }${diffValue} points")`
+  } else if (metricType === 'lcp' || metricType === 'tbt') {
+    return `[${currentValue} ms ${isRegression ? '🔴' : '🟢'}](## "The ${metricType} has ${
+      isRegression ? 'increased in +' : 'decreased in '
+    }${diffValue} ms")`
+  } else if (metricType === 'cls') {
+    return `[${currentValue} ${isRegression ? '🔴' : '🟢'}](## "The CLS has ${
+      isRegression ? 'increased in +' : 'decreased in'
+    } ${diffValue}")`
+  } else {
+    return ''
+  }
```

Comment:
If you always return inside an `if` you don't need and `else if` later on, just `if` is enough

Reply:
Ahaha yes that's true, but some things are hard to let go.

I will take a look in another PR, because I think this could be better handled with a switch instead of this many ifs.

What do you think?

Reply:
Perfect